modules/orchestrator/src/main/java/org/eclipse/xpanse/modules/orchestrator/PluginManager.java

```diff
@@ -46,17 +46,20 @@ public void loadPlugins() {
         applicationContext.getBeansOfType(OrchestratorPlugin.class).forEach((key, value) -> {
             if (isPluginUsable(value.requiredProperties(), value.getCsp())) {
                 pluginsMap.put(value.getCsp(), value);
-            } else {
-                log.error(String.format(
-                        "Plugin for csp %s is not in usable state. Will not be activated.",
-                        value.getCsp()));
             }
         });
         checkCredentialDefinitions();
-        log.info("Cloud service providers:{} with activated plugins.", pluginsMap.keySet());
+        log.info("Cloud service providers {} with activated plugins.", pluginsMap.keySet());
         if (pluginsMap.isEmpty()) {
             throw new PluginNotFoundException("No plugin is activated.");
         }
+        if (pluginsMap.size() > 1) {
+            String osAuthUrl = applicationContext.getEnvironment().getProperty("OS_AUTH_URL");
+            if (Objects.nonNull(osAuthUrl)) {

```

Comment:
The logic here is fine. But we need another way to get the list of of "black listed" environment variables. 
We can have this is a list in properties file.